samples/src/Microsoft.Azure.IIoT.App/src/Models/CredentialModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+using System.ComponentModel.DataAnnotations;
+
+namespace Microsoft.Azure.IIoT.App.Models {
+    public class CredentialModel {
+        [Required]
+        [StringLength(25, MinimumLength = 3, ErrorMessage = "UserName should be between 3 and 25 characters.")]
+        public string Username { get; set; }
```

Comment:
I think we should not add any restrictions for username nor for password. you get all kinds of servers  weird in the field ...

Reply:
ok removed